### DISCUSSION TITLE
Hey Snolting do you plan to attend FPGA Ignite 2024 in Heidelberg?

### DISCUSSION THREAD

**Discussion by ucycg** [stnolting/neorv32#946] [Q&A] [answered]
Hey Stephan,

sry in advance I hope this doesn't sound to much like a crappy advertisement :100: 

Are you aware of **FPGA Ignite Summer School 2024 in Heidelberg**? ([https://fpga-ignite.github.io](url)) 

**"The summer school is a one week opportunity for networking, exciting lectures and a hackathon on designing a custom RISC-V SoC (that will be subsequently taped-out)."**

I ask because I plan to attend myself and I'm already successfully registered. I'd love to have the chance to meet you in person and to talk about Neorv32 if possible.

On the other hand I think you would be a valuable participant with your extensive knowledge about RISC-V and its a great platform to present the NeoRV32 to a very suitable audience. Maybe you could also profit  from some of the workshops i.e. the cocotb verfication? Maybe it accelerates the verification process of the NeoRV32? 

The event takes place from August 5th to 9th in Heidelberg at the ZITI ([http://www.ziti.uni-heidelberg.de/de/](url)). It is organized by Prof Dirk Koch and his group of Novel Computing Technologies ([https://www.ziti.uni-heidelberg.de/de/forschung/nct.html](url)). It will take place for the 3rd time by now since the inauguration of Prof Koch as Full Professor in April 2022. 

Online Participation is also possible 

Kind regards

Justin

**Comment by ucycg** [comment]
P.S.: How do I make the links work? I think its because they are linking to websites outside of github. Sry I didnt take the time to solve this I hope you still take a look at the websites :)

**Reply by stnolting** [reply]
That's just Markdown:
```
[some text](address)
```

Example:
```
[NEORV32](https://github.com/stnolting/neorv32)
```
Renders into:
> [NEORV32](https://github.com/stnolting/neorv32)

**Reply by ucycg** [reply]
Ty very much :)

**Comment by stnolting** [comment, marked as answer]
Hey Justin.

No, I am not planning to go there (too much work over here 😅). A colleague of mine was there last year and thought the workshop was pretty awesome.

However, I'm quite familiar with the topics / the workshop and especially with Uni Heidelberg an Dirk Koch himself. We are cooperating in some projects. :wink: Beyond that, I finally had the change to meet him and one of his colleague (who is also supervising the workshop) in person at the RISC-V Summit in Munich two weeks ago. I think the workshop will be a good experience where one can learn a lot!

And just to do some _more_ advertising: Fraunhofer is also preparing a RISC-V hackathon 😎

**Reply by ucycg** [reply]
Hey, ty for the detailed answer! Its great to hear that. Its understandable that you are busy :)
